integration_tests/src/main/python/delta_lake_update_test.py

```diff
@@ -26,18 +26,18 @@
                                              "spark.rapids.sql.command.UpdateCommandEdge": "true"})
 
 def delta_sql_update_test(spark_tmp_path, use_cdf, dest_table_func, update_sql,
-                          check_func, partition_columns=None, enable_deletion_vectors=False):
+                          check_func, partition_columns=None, enable_deletion_vectors=False, conf=None):
     data_path = spark_tmp_path + "/DELTA_DATA"
     def setup_tables(spark):
-        setup_delta_dest_tables(spark, data_path, dest_table_func, use_cdf, partition_columns, enable_deletion_vectors)
+        setup_delta_dest_tables(spark, data_path, dest_table_func, use_cdf, enable_deletion_vectors, partition_columns)
```

Comment:
Is the change here intentional? Did we really switch the order of deletion vectors and partition columns just because?